### DISCUSSION TITLE
How can we login in self host runner using organization credentials?

### DISCUSSION THREAD

**Discussion by in8makup** [community/community#25060] [Actions]
<p>I am running self hosted runner but due to credetials issue , application is not opening. I am getting logon error meaasage while opening the application URL. This is working fine in my local.</p>

**Comment by Yanjingzhu** [comment]
<p>For me to better understand your scenario, could you answer my next questions?  </p>
<ol>
<li>What’s the organization credentials ? Your loacl domain user or github organization user?  </li>
<li>What’s the opreating system of the machine which hold your self-hosted runner? Windows, Linux or MacOs?</li>
<li>Do you run your runner as <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-the-self-hosted-runner-application-as-a-service" rel="nofollow noopener">service</a> or in interactive mode?  </li>
<li>
</li>
</ol>
<hr>
<p> I am getting logon error meaasage while opening the application URL. </p>
<hr>
<p>How do you open and logon to your application? Can you share your workflow yml content?  </p>

**Comment by in8makup** [comment]
<ol>
<li>
<p>My system user name and password(local user). My user name is assigned to the Application(SAP). It should Open with SSO login but its throughing error “Logon Error Message”</p>
</li>
<li>
<p>Windows 10</p>
</li>
<li>
<p>I am running as service</p>
</li>
<li>
<p>I am opening application through Selenium in IE. In my local its not asking any credentils. Is there any way we can login with system credentials(my local credentials) using self hosted runner?</p>
</li>
</ol>
<p>name: Selenium1 CI</p>
<p>on: [push]</p>
<p>jobs:<br>
build:</p>
<p>runs-on: [self-hosted]</p>
<p>steps:</p>
<ul>
<li>uses: actions/checkout@master</li>
<li>name: Set up JDK 1.8<br>
uses: actions/setup-java@master<br>
with:<br>
java-version: 1.8</li>
<li>name: Setup Maven<br>
uses: stCarolas/setup-maven@v1</li>
<li>name: Build with Maven<br>
run: mvn -B package --file pom.xml</li>
</ul>
<hr>
<p>Did I miss anything here?</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Which account do you use to run the runner service? If it is not your local user account, please change to it. </p>
<p>When configure a runner to run as service, you will be asked to input user account to use for the service. </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1cfb132156a6aff544d0e540bbbba904382a5b9e.png" data-download-href="/uploads/short-url/48nifgNFjGHKFWZNARVQqmNuMVM.png?dl=1" title="runner service account.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181088613-7db0a677-68eb-4c0f-a6aa-948d01ed7243.png" alt="runner service account.png" data-base62-sha1="48nifgNFjGHKFWZNARVQqmNuMVM" width="690" height="342" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1cfb132156a6aff544d0e540bbbba904382a5b9e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">runner service account.png</span><span class="informations">889×441 11.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If you are running UI tests such as Selenium or Coded UI tests that require a browser, the browser is launched in the context of the this runner account. </p>
<p>Also, you could see which user account is used by the service in the Services . You could try to change the <strong>Log On As</strong> by right clicking on the service &gt; choose Properties Option&gt; choose Log On tab, change the account to your local user and then restart the runner service.  </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/51035774f205d73c0f6839b3f2676a498d059353.png" data-download-href="/uploads/short-url/byFPJRwcvn2Yi6V3yvW7lnfDDph.png?dl=1" title="logon as.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181088615-6fabf347-ac88-41f5-8a6c-697aced9b586.png" alt="logon as.png" data-base62-sha1="byFPJRwcvn2Yi6V3yvW7lnfDDph" width="690" height="98" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/51035774f205d73c0f6839b3f2676a498d059353_2_690x98.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/51035774f205d73c0f6839b3f2676a498d059353_2_1035x147.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/5/51035774f205d73c0f6839b3f2676a498d059353_2_1380x196.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/51035774f205d73c0f6839b3f2676a498d059353_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">logon as.png</span><span class="informations">1697×242 26.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p><img src="https://user-images.githubusercontent.com/24806478/181088617-45e5357d-9eb0-47ac-806c-88e42e6f3a90.png" alt="logon as service.png" data-base62-sha1="9DCp9WT43J7A9qkCwubsP4TkklA" width="409" height="473"></p>
